analysis/pioneer_opt_in_estimates.lua

```diff
@@ -0,0 +1,56 @@
+--[[
```

Comment:
Please use the standard header with MD documentation as seen here https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/heka/sandboxes/heka/analysis/heka_message_schema.lua

This will auto publish to gitbook like this: https://mozilla-services.github.io/lua_sandbox_extensions/heka/sandboxes/heka/analysis/heka_message_schema.html